src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterEscapeChar.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
+
+/*
+ * Config: Default.
+ */
+public class InputSuppressionXpathFilterEscapeChar { // violation
+
+    char a = '&'; // violation
+
+    char b = '\"';
+
+    char c = '\'';
+
+    char d = '<';
+
+    char e = '>';
+    // violation
```

Comment:
this violation comment looks weird. Please place comments correctly (to each line with violations)

Reply:
Done.